manifests/m/Mikrotik/Winbox/3.40/Mikrotik.Winbox.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created with komac v2.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Mikrotik.Winbox
+PackageVersion: '3.40'
+InstallerType: exe

```

```suggestion
InstallerType: portable
```
From the image above, I'm guessing this package is of portable type.